interface-definitions/system_option.xml.in

```diff
@@ -37,7 +37,157 @@
+                      <help>Disable the NMI watchdog for detecting hard CPU lockups</help>
+                      <valueless/>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="isolate-cpus">
+                    <properties>
+                      <help>Isolate specified CPUs from the scheduler</help>
+                      <valueHelp>
+                        <format>u32:0-511</format>
+                        <description>CPU core</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>&lt;start-end&gt;</format>
+                        <description>CPU cores range (e.g. 4-7)</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format> </format>
+                        <description>\n\n  Multiple CPUs can be specified as a comma-separated list.\n  For example: '1,2,5-7'</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="cpu"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="nohz-full">
+                    <properties>
+                      <help>Enable full tickless mode for specified CPUs</help>
+                      <valueHelp>
+                        <format>u32:0-511</format>
+                        <description>CPU core</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>&lt;start-end&gt;</format>
+                        <description>CPU cores range (e.g. 4-7)</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format> </format>
+                        <description>\n\n  Multiple CPUs can be specified as a comma-separated list.\n  For example: '1,2,5'</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="cpu"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="rcu-no-cbs">
+                    <properties>
+                      <help>Offload Read-Copy-Update (RCU) callback processing to specified CPUs</help>
+                      <valueHelp>
+                        <format>u32:0-511</format>
+                        <description>CPU core</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>&lt;start-end&gt;</format>
+                        <description>CPU cores range (e.g. 4-7)</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format> </format>
+                        <description>\n\n  Multiple CPUs can be specified as a comma-separated list.\n  For example: '1,2,5'</description>
```

Comment:
If this is a necessary fixup for some display issue, we need to fix its root case, not add workarounds.

Reply:
Attached the file without this workaround
![cpu](https://github.com/user-attachments/assets/b4d12beb-04ac-4782-b1a1-27463d2e5106)

Reply:
Description updted to
```
vyos@r14# set system option kernel cpu isolate-cpus 
Possible completions:
   <0-511>              CPU core
   <start-end>          CPU core range (examples: "1", "4-7", "1,2-5,7")
                        

      
[edit]
vyos@r14# 
```